src/pages/registration/contributors_tab/components/ContributorAuthors.tsx

```diff
@@ -0,0 +1,42 @@
+import { ErrorMessage, FieldArray, FieldArrayRenderProps, useFormikContext } from 'formik';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { FormHelperText, MuiThemeProvider, Typography } from '@material-ui/core';
+import BackgroundDiv from '../../../../components/BackgroundDiv';
+import lightTheme from '../../../../themes/lightTheme';
+import { ContributorFieldNames } from '../../../../types/publicationFieldNames';
+import { Registration } from '../../../../types/registration.types';
+import Authors from '../Authors';
+
+export const ContributorAuthors = () => {
+  const { t } = useTranslation('registration');
+  const {
+    values: {
+      entityDescription: { contributors },
+    },
+    errors,
+    touched,
+  } = useFormikContext<Registration>();
+  const contributorsError = errors.entityDescription?.contributors;
+  const contributorsTouched = touched.entityDescription?.contributors;
+
+  return (
+    <>
+      <BackgroundDiv backgroundColor={lightTheme.palette.section.main}>
+        <Typography variant="h2">{t('contributors.authors')}</Typography>
+        <FieldArray name={ContributorFieldNames.CONTRIBUTORS}>
+          {({ push, replace }: FieldArrayRenderProps) => (
+            <MuiThemeProvider theme={lightTheme}>
+              <Authors push={push} replace={replace} />
+            </MuiThemeProvider>
+          )}
+        </FieldArray>
+      </BackgroundDiv>
+      {contributors.length === 0 && !!contributorsTouched && typeof contributorsError === 'string' && (
+        <FormHelperText error>
+          <ErrorMessage name={ContributorFieldNames.CONTRIBUTORS} />
+        </FormHelperText>
+      )}
+    </>
```

Comment:
Det er ganske mye likt i returnen her og den i ContributorEditors. Men vet ikke om det er verdt det å lage enda en komponent for så å sende inn props? eller om det er greit at det er litt likt?

Reply:
Det var litt krøll tidligere med feilmelding her i hvert fall, så hadde vært greit å bare kunne forholde seg til en komponent til det (kommer vel flere typer enn forfatter og redaktør også). hvis det er relativt greit å få til iaff

Reply:
tar dette i egen task: https://unit.atlassian.net/browse/NP-2210